### PR TITLE
feat(kousa,docs): make it easier to set environment variables by adding scripts.

### PR DESCRIPTION
**Pull request by samarmohan**
Create two files (one for windows, one for unix-based) with all the environment variables needed and
update CONTRIBUTING.md accordingly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -123,25 +123,7 @@ $ CREATE DATABASE kousa_repo2;
...
-export TWITTER_BEARER_TOKEN=
-export GITHUB_CLIENT_SECRET=
-export SENTRY_DNS=
-export API_URL=http://localhost:4001
-export WEB_URL=http://localhost:3000
-export PORT=4001
-```
-
-> You can save these variables in a `.txt` and run `source path/to/file.txt`
+Navigate to `/kousa` and run `call env.windows.bat` if you are on Windows or `source env.unix.sh` if you are on a Mac or Unix based system. This will set the necessary environment variables.
```
Comment (amitojsingh366) [inline CONTRIBUTING.md:126]: this is not what we want, the env variables shown here are just examples and need to be filled... we don't want to supply the contributors with our own versions of them.

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (samarmohan) [reply CONTRIBUTING.md:126]: What should I do then?

Comment (amitojsingh366) [reply CONTRIBUTING.md:126]: leave it as is....

**Comment by samarmohan** [comment]
Anything stopping this from being merged

**Comment by samarmohan** [comment]
sorry that was a test using GH cli

**Pull request closed without merging**